### PR TITLE
Added a tip to the unit test README

### PR DESCRIPTION
**Pull request by xarkes**
Help users run a specific testcase rather than the whole testsuite.

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -38,9 +39,11 @@ can automatically fix the test so that it matches the new output (if that is the
 right behaviour!) or it can mark it as broken for you.
 
 ## Unit tests
+
 To run unit tests, just use `ninja -C build test` (or `meson test -C build`)
 from the top directory (replace `build` with the name of the directory you used
 to build Rizin).
+You can run one specific testcase (e.g. `bin`) using `meson test -C build bin`.
```
Comment (XVilka) [inline test/README.md]: Probably a "testcase category" instead?

**Review by XVilka — CHANGES_REQUESTED** [review]

File: test/README.md (hunk 1 above)
Comment (wargio) [reply test/README.md]: i think is ok there. let's see what the others thinks about it.

Comment (ret2libc) [reply test/README.md]: EIther way is fine for me.

Comment (ITAYC0HEN) [reply test/README.md]: I am with @XVilka, "testcase category" is better imo

**Review by ITAYC0HEN — CHANGES_REQUESTED** [review]

**Review by ret2libc — APPROVED** [review]

**Review by wargio — APPROVED** [review]

**Review by XVilka — APPROVED** [review]

**Pull request merged**